src/utils/tickets.js

```diff
@@ -42,6 +88,7 @@ const processTicket = async ticket => {
   return {
     success: true,
     fileArtifacts: artifacts.map(ArtifactManager.artifactToPath),
+    malwareMatches: JSON.stringify(await malwareMatches),
```

Comment:
Move `await` keyword to where the function call is made on L57, it’s weird to leave the promise sitting around until the end.

Reply:
`await` will cause pause on the current thread, so that await here should make the function run in parallel. (?)

Reply:
It pauses three times regardless of where that await is - once for each `await`. It's just more confusing to have it there. I agree that it'd be nice to have everything done in parallel - you could use [`Promise.all`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) to accomplish that. It might require a bit more work to ensure that the multiple managers trying to manipulate the browser instance don't conflict with each other.

Reply:
We can discuss this one at meeting, maybe push a commit before merging.

Reply:
Sure thing.